### ISSUE TITLE
type only props parsing

### ISSUE THREAD

**Issue by Disservin** [tkskto/vue-component-analyzer#308]
Hi, 

I am fairly certain your props parsing fails for vue's new way of defining props.

```
const props = defineProps<{
  foo: string
  bar?: number
}>()
```

**Comment by Disservin** [comment]
You might also want to consider using https://github.com/vuejs/language-tools/tree/master/packages/vue-component-meta ;)

**Comment by tkskto** [comment]
@Disservin I'm really sorry for the delay in responding to your issue.

I published [v0.9.0](https://github.com/tkskto/vue-component-analyzer/releases/tag/v0.9.0), and this issue should be solved. If not, please let me know again🙏

> You might also want to consider using https://github.com/vuejs/language-tools/tree/master/packages/vue-component-meta ;)

Thank you for your suggestion. However, it seemed a bit complicated, so I will try it in the next version.

**Comment by Disservin** [comment]
No problem :D  
To be honest I can't even remember how I ended up here because I wanted to extract the prop types from a component which isn't the use case for your library :D component-meta helped me a bit there i.e.
```ts
const tsconfigChecker = createComponentMetaChecker(
    path.join(dirname, "./tsconfig.json"),
    checkerOptions,
);

const meta = tsconfigChecker.getComponentMeta(filePath);
```
and over `meta.props` you can access the different types of the props :D
